Solutions/vArmour/Analytic Rules/vArmourApplicationControllerSMBRealmTraversal.yaml

```diff
@@ -0,0 +1,22 @@
+id: a36de6c3-3198-4d37-92ae-e19e36712c2e
+name: vArmour AppController - SMB Realm Traversal
+description: |
+  'Detects when SMB traffic crosses Production and Non-Production Realms.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: vArmourAC
+    dataTypes:
+      - "CommonSecurityLog (vArmour)"
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Collection
+  - LateralMovement
+query: |
+  CommonSecurityLog
+  | where DeviceVendor == "vArmour"
+  | where DeviceProduct == "AC"
+  | where Activity == "POLICY_VIOLATION"
+  | extend PolicyName = extract("(SMB_REALM_TRAVERSAL\\w+{.*})", 1, DeviceCustomString1)
```

Comment:
Should have minimum of 1 entity mapping. Please check other analytic rules files https://github.com/Azure/Azure-Sentinel/tree/master/Detections/CommonSecurityLog for reference.

Reply:
Added host and IP entities